pootle/apps/pootle_project/models.py

```diff
@@ -181,7 +181,7 @@ class Project(models.Model, CachedTreeItem, ProjectURLMixin):
                             db_index=True, verbose_name=_('Code'), blank=False,
                             help_text=code_help_text)
 
-    fullname = models.CharField(max_length=255, null=False,
+    fullname = models.CharField(max_length=255, null=False, blank=True,
```

Comment:
I don't think making `fullname` optional was needed or agreed upon, but rather ensuring it was non-empty.